Documentation/Signoffs/Camera_Power_Signoff.md

```diff
@@ -0,0 +1,136 @@
+# Primary Data Acquisition Power Detail Design
+
+## Big Picture
+
+The primary data acquisition subsystem will utilize a number of different electrical components in order to ensure the cameras can transfer images over to the server. Given its high elevation location on the outdoor posts, there will be no easily feasible way to wire the subsystem to a grounded power source. Therefore, a “wireless” solution will have to be developed in order to ensure that the cameras can have a reliable and hands-off way to remain powered, along with any other needed devices to ensure the system remains operational.
+
+## Specifications
+
+Hardware Constraints
+
+1. Avigilon Cameras [1]
+    * Input Voltage - 12 VDC +/- 10%
+    * Input Power - 6 W min 
+    * Power Supply Connection - Power over Ethernet (IEEE 802.3af Class 3), 2-pin terminal block
+2. WiFi Module [2]
+    * Input Voltage - 5 VDC (Max 24 VDC)
+    * Current Draw - 2 A
+    * Power Supply Connection - USB, 2.1mm plug
+3. Arduino Nano 33 IoT [3]
+    * Input Voltage - 3.3 V (Max 21 V)
+    * Current Draw - 85 mA [4]
+    * Power Supply Connection - USB, Vin & GND pin
+4. Heater Plate [5]
+    * Input Voltage - 5 V
+    * Input Power - 1 W
+    * Power Supply Connection - Positive and Negative wire
+
+Reliability Constraints
+
+1. Should keep all devices powered during operational hours (8:00am - 4:30pm)
+2. Should not interfere with operations of the primary data acquisition system
+3. Power source should be rechargeable to minimize maintenance
+
+## Buildable Schematic
+
+![PDA_Wiring_Schematic](https://user-images.githubusercontent.com/80428236/219553780-1b5e38c1-d0f1-47f3-a0a0-48e2fc9916c7.png)
+
+Figure 1. Electrical Wiring Schematic
+
+## Analysis
+
+Selecting the Power Source
+
+1. It serves to the convenience of this subsystem that all devices that need to be powered can accept or require 12 V to operate (as seen in the constraints section). Therefore, it has been decided to go with a 12 V 7 Ah LiFePO4 battery produced by Howell Energy [6]. These batteries do not occupy much space (5.94 x 2.56 x 3.7 in) and last much longer than their sealed lead-acid counterpart, as well as being capable of being recharged [6]. 
```

Comment:
This is incorrect based on the constraints. Also, eliminate the unnecessary fluff.